confgenerator/confgenerator.go

```diff
@@ -383,7 +383,7 @@ func (l *Logging) generateFluentbitComponents(userAgent string, hostInfo *host.I
 		BufferInMemory: true,
 	}.Components("ops-agent-fluent-bit")...)
 
-	out = append(out, stackdriverOutputComponent("ops-agent-fluent-bit", userAgent))
+	out = append(out, stackdriverOutputComponent("ops-agent-fluent-bit", userAgent, ""))
```

Comment:
Why aren't we also setting a limit for self-logs?

Reply:
Self logs don't buffer on disk, so this configuration does not have effect.